request_a_govuk_domain/request/templates/404.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% load govuk_frontend_django %}
+
+{% block title %}Page not found{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-xl">
+        Page not found
+      </h1>
+
+      <p class="govuk-body">
+        If you entered a web address, check it is correct.
+      </p>
+
+      <p class="govuk-body">
+        You can <a class="govuk-link" href="/">go back to the service's start page</a>.
```

Comment:
maybe "home page"? no strong opinion

Reply:
Changed. I think it's going to be rewritten but no strong opinion either.